engine/src/main/java/org/camunda/bpm/engine/repository/DecisionDefinitionQuery.java

```diff
@@ -193,6 +193,12 @@
    * Note that the ordering of decision definitions without tenant id is database-specific. */
   DecisionDefinitionQuery orderByTenantId();
 
+  /** Order by key of the decision requirements definition a decision definition belongs to
```

Comment:
Seems like this is out of place:
```suggestion
  /** Order by key of the decision requirements definition 
```